src/analyses/invariant_set.cpp

```diff
@@ -511,48 +509,49 @@ void invariant_sett::strengthen_rec(const exprt &expr)
     // special rule: x = (a & b)
     // implies:      x<=a && x<=b
 
-    if(expr.op1().id()==ID_bitand)
+    if(equal_expr.op1().id() == ID_bitand)
     {
-      const exprt &bitand_op=expr.op1();
+      const exprt &bitand_op = equal_expr.op1();
 
       forall_operands(it, bitand_op)
       {
-        exprt tmp(expr);
+        exprt tmp(equal_expr);
         tmp.op1()=*it;
         tmp.id(ID_le);
         strengthen_rec(tmp);
       }
 
       return;
     }
-    else if(expr.op0().id()==ID_bitand)
+    else if(equal_expr.op0().id() == ID_bitand)
     {
-      exprt tmp(expr);
+      exprt tmp(equal_expr);
       std::swap(tmp.op0(), tmp.op1());
       strengthen_rec(tmp);
       return;
     }
 
     // special rule: x = (type) y
-    if(expr.op1().id()==ID_typecast)
+    if(equal_expr.op1().id() == ID_typecast)
     {
-      assert(expr.op1().operands().size()==1);
-      add_type_bounds(expr.op0(), expr.op1().op0().type());
+      assert(equal_expr.op1().operands().size() == 1);
+      add_type_bounds(equal_expr.op0(), equal_expr.op1().op0().type());
     }
-    else if(expr.op0().id()==ID_typecast)
+    else if(equal_expr.op0().id() == ID_typecast)
     {
-      assert(expr.op0().operands().size()==1);
-      add_type_bounds(expr.op1(), expr.op0().op0().type());
+      assert(equal_expr.op0().operands().size() == 1);
+      add_type_bounds(equal_expr.op1(), equal_expr.op0().op0().type());
     }
 
     std::pair<unsigned, unsigned> p, s;
 
-    if(get_object(expr.op0(), p.first) ||
-       get_object(expr.op1(), p.second))
+    if(
+      get_object(equal_expr.op0(), p.first) ||
+      get_object(equal_expr.op1(), p.second))
```

Comment:
Nit pick: please add braces for readability.

Reply:
done